.github/workflows/actions-self-test.yml

```diff
@@ -4,15 +4,15 @@ on:
   push:
 
 env:
-  PRODUCT_VERSION: "1.8.6"
+  PRODUCT_VERSION: "1.10.0"
```

Comment:
The latest stable release is `1.10.3` according to `hcv`:

>  $ hcv packer list
> 1.10.3

Reply:
It is indeed. Normally I'd update it to the patch level, but I think keeping it on the minor is good enough